src/js/webrequest.js

```diff
@@ -706,6 +715,7 @@ function startListeners() {
   chrome.webRequest.onHeadersReceived.addListener(onHeadersReceived, {urls: ["<all_urls>"]}, ["responseHeaders", "blocking"]);
   chrome.tabs.onRemoved.addListener(onTabRemoved);
   chrome.tabs.onReplaced.addListener(onTabReplaced);
+  chrome.tabs.onUpdated.addListener(onTabUpdated);
```

Comment:
Yeah, we actually have some duplications still of the `chrome.tabs` listeners. To be fair, I'm really not helping the cause here though... :P 

- `chrome.tabs.onRemoved` appears in webrequest.js and incognito.js
- `chrome.tabs.onReplaced` appears in webrequest.js and background.js
- `chrome.tabs.onUpdated` appeared in background.js and incognito.js (and now in webrequest.js).

I wasn't too worried about these, as they're very infrequent in terms of the frequency at which they're called, but it shouldn't be too much trouble to merge them all. Should I go ahead and do this?